streaming/multimodal/__init__.py

```diff
@@ -3,4 +3,6 @@
 
 """Natively supported multimodal datasets."""
 
-from streaming.multimodal.webvid import StreamingWebVid as StreamingWebVid
+from streaming.multimodal.webvid import StreamingInsideWebVid as StreamingInsideWebVid
```

Comment:
Wondering, why not just `from streaming.multimodal.webvid import StreamingInsideWebVid` ?

Reply:
linting